### PR TITLE
[#40424] mail schedule booking for partner in lastweek

### PR DESCRIPTION
**Pull request by trunghq-2593**
Screenshot from 2021-08-24 18-28-34

## Related Tickets
- [#40424](https://edu-redmine.sun-asterisk.vn/issues/40424)

## WHAT (optional)
- Send mail schedule

## HOW
- Command (Task Schedule)
- Mail Trap
- Mindset
+ get total revenues
+ get total order
+ get data of detail order ( info of order & info User)
+ show data and send mail

### CODE REVIEW

**Comment by trunghq-2593** [comment]
ready

File: app/Console/Kernel.php [hunk 1]
```diff
@@ -24,7 +24,7 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        // $schedule->command('inspire')->hourly();
+         $schedule->command('mail:command')->weeklyOn(0, '20:00');
```
Comment (ghost) [inline app/Console/Kernel.php]: bỏ time vào config nha

Comment (trunghq-2593) [reply app/Console/Kernel.php]: oke b

**Comment by trunghq-2593** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**